mkosi.md

```diff
@@ -1140,196 +1140,208 @@ under the assumption that it is invoked from a *source*
+  `openmandriva`) for the image, or additional distribution packages
+  to install. Note that all options encoded in this configuration file
+  may also be set on the command line, and this file is hence little
+  more than a way to make sure invoking `mkosi` without further
+  parameters in your *source* tree is enough to get the right image of
+  your choice set up.
+
+  Additionally, if a *`mkosi.default.d/`* directory exists, each file in
+  it is loaded in the same manner adding/overriding the values
+  specified in `mkosi.default`. If `mkosi.default.d/` contains a
   directory named after the distribution being built, each file in
-  that directory is also processed. The file format is inspired by
-  Windows `.ini` files and supports multi-line assignments: any line
-  with initial whitespace is considered a continuation line of the line
-  before. Command-line arguments, as shown in the help description,
-  have to be included in a configuration block (e.g.  "`[Packages]`")
-  corresponding to the argument group (e.g. "`Packages`"), and the
-  argument gets converted as follows: "`--with-network`" becomes
-  "`WithNetwork=yes`". For further details see the table above.
-
-* `mkosi.extra/` or `mkosi.extra.tar` may be respectively a directory
-  or archive. If any exist all files contained in it are copied over
-  the directory tree of the image after the *OS* was installed. This
-  may be used to add in additional files to an image, on top of what
-  the distribution includes in its packages. When using a directory
-  file ownership is not preserved: all files copied will be owned by
-  root. To preserve ownership use a tar archive.
-
-* `mkosi.skeleton/` or `mkosi.skeleton.tar` may be respectively a
-  directory or archive, and they work in the same way as
-  `mkosi.extra`/`mkosi.skeleton.tar`. However the files are copied
-  before anything else so to have a skeleton tree for the OS. This
-  allows to change the package manager and create files that need to
-  be there before anything is installed. When using a directory file
-  ownership is not preserved: all files copied will be owned by
-  root. To preserve ownership use a tar archive.
-
-* `mkosi.build` may be an executable script. If it exists the image
+  that directory is also processed.
+
+  The file format is inspired by Windows `.ini` files and supports
+  multi-line assignments: any line with initial whitespace is
+  considered a continuation line of the line before. Command-line
+  arguments, as shown in the help description, have to be included in
+  a configuration block (e.g.  "`[Packages]`") corresponding to the
+  argument group (e.g. "`Packages`"), and the argument gets converted
+  as follows: "`--with-network`" becomes "`WithNetwork=yes`". For
+  further details see the table above.
+
+* **`mkosi.skeleton/`** directory or **`mkosi.skeleton.tar`** archive
+  are used to insert files into the image. The files are copied
```

Comment:
"are used" -> "may be used", otherwise it's unclear from the paragraph, whether these are mandatory or not.